python/ray/train/tests/test_worker_group.py

```diff
@@ -59,6 +76,44 @@ def test_worker_restart(ray_start_2_cpus):
     wg.execute(lambda: 1)
 
 
+def test_worker_with_gpu_ids(ray_start_2_cpus_and_gpus):
+    num_gpus = 2
+    wg = WorkerGroup(num_workers=2, num_gpus_per_worker=1)
+    assert len(wg.workers) == 2
+    time.sleep(1)
+    assert ray_constants.GPU not in ray.available_resources()
+    wg.execute(lambda: 1)
+    assert len(wg.workers) == 2
+    for w in wg.workers:
+        gpu_and_accelerator_ids = w.metadata.resource_ids
+        assert len(gpu_and_accelerator_ids) == 2
```

Comment:
Looks like this is [failing](https://buildkite.com/ray-project/oss-ci-build-pr/builds/35951#018a8a46-24c3-44c2-990c-b77e198e28dd) right now.